### DISCUSSION TITLE
Is there a way to change the key shortcut for choosing from lisp-implementations?

### DISCUSSION THREAD

**Discussion by mdbergmann** [joaotavora/sly#509] [Q&A] [answered]
According to the documentation the key shortcut for this is by prefixing M-x with M--. 
But M-- I've bound to something else.

**Comment by monnier** [comment, marked as answer]
> According to the documentation the key shortcut for this is by prefixing M-x with M--. 
> But M-- I've bound to something else.

You can use `C--` and `C-u -`, I think.

**Reply by mdbergmann** [reply]
Yep, `C--` also works. Thanks.
